Rubberduck.Core/UI/CodeExplorer/Commands/ImportCommand.cs

```diff
@@ -562,6 +571,18 @@ protected override void OnExecute(object parameter)
 
         private string FilterExtension => string.Join("; ", ImportableExtensions.Select(ext => $"*{ext}"));
 
+        private string AllNonDocumentModulesExtension => string.Join("; ",
+            ImportableExtensions.Where(ext => !ext.EndsWith("doccls"))
+                                .Select(ext => $"*{ext}"));
+
+        private string StandardModuleExtension => "*" + ImportableExtensions.Where(ext => ext.EndsWith("bas")).First();
```

Comment:
My concern with using `First()` is that it becomes a runtime error that will blow upmif the assumption about file extensions being unique and present is violated.

I probably would use ‘FirstOrDefault()` and if `null`, do not concatenate into the filter. Then I would add an unit test to assert that the contents are unique and has valid extensions, thus preventing a time bomb.